integration-test/1491-france-shields.py

```diff
@@ -0,0 +1,255 @@
+            dsl.relation(1, {
+                'name': 'E 401 Saint-Brieuc - Caen', 'type': 'route',
+                'route': 'road', 'source': 'openstreetmap.org',
+                'name:fr': u'Route europ\xe9enne E 401',
+                'wikidata': 'Q1922885', 'ref': 'E 401',
+                'description:fr': 'E 401 Saint-Brieuc - Caen',
+                'network': 'e-road',
+            }, ways=[147156883]),
+            dsl.relation(2, {
+                'name:en': 'E 03 Cherbourg - La Rochelle',
+                'name': u'Route europ\xe9enne E 03', 'type': 'route',
+                'route': 'road', 'name:cs': u'Evropsk\xe1 silnice E03',
+                'name:fr': u'Route europ\xe9enne E 03',
+                'source': 'openstreetmap.org',
+                'e-road:class': 'A-intermediate',
+                'wikipedia': 'en:European route E03',
+                'wikidata': 'Q585439', 'ref': 'E 03',
+                'description:fr': 'E 03 Cherbourg - La Rochelle',
+                'network': 'e-road',
+            }, ways=[147156883]),
+            dsl.relation(3, {
+                'name': 'Autoroute des Estuaires', 'ref': 'A 84',
+                'route': 'road', 'source': 'openstreetmap.org',
+                'type': 'route', 'network': 'FR:A-road',
+            }, ways=[147156883]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 147156883,
+                'shield_text': 'A84', 'network': 'FR:A-road',
+                'all_networks': ['FR:A-road', 'e-road', 'e-road'],
+                'all_shield_texts': ['A84', 'E3', 'E401'],
+            })
+
+    def test_fr_m_road(self):
+        import dsl
+
+        z, x, y = (16, 34078, 23914)
+
+        self.generate_fixtures(
+            dsl.is_in('FR', z, x, y),
+            # https://www.openstreetmap.org/way/4117272
+            dsl.way(4117272, dsl.tile_diagonal(z, x, y), {
+                'maxspeed': '70', 'bicycle': 'no',
+                'name': 'Promenade Edouard Corniglion-Molinier',
+                'surface': 'asphalt', 'source': 'openstreetmap.org',
+                'oneway': 'yes', 'ref': 'M 6098', 'highway': 'primary',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 4117272,
+                'shield_text': 'M6098', 'network': 'FR:M-road',
+            })
+
+    def test_fr_d_bis_road(self):
+        import dsl
+
+        z, x, y = (16, 33472, 22990)
+
+        self.generate_fixtures(
+            dsl.is_in('FR', z, x, y),
+            # https://www.openstreetmap.org/way/20225575
+            dsl.way(20225575, dsl.tile_diagonal(z, x, y), {
+                'source': 'openstreetmap.org', 'ref': 'D 977bis',
+                'highway': 'secondary',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 20225575,
+                'shield_text': 'D977bis', 'network': 'FR:D-road',
```

Comment:
Does the `bis` represent "express"? Do we need to somehow differentiate those shields from normal D-roads?

Reply:
A random site [about driving in France] says that "bis" routes are "secondary" routes. Apparently, they're the opposite of express, and closer in meaning to "scenic"!

What would make sense here:

* `shield_text: D977`, `network: FR:D-road:scenic`? Or,
* `shield_text: D977bis`, `network: FR:D-road:scenic`? Or,
* something else?